server/views/pages/search/caSearch/caSearch.njk

```diff
@@ -288,7 +288,7 @@
 {% block content %}
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-three-quarters">
-            <input type="hidden" id="searchTerm" value="{{ queryTerm }}">
+            <input type="hidden" id="searchTerm" value="{{ queryTerm | titlecase}}">
```

Comment:
added titlecase formatting so that the user input if all lowercase/uppercase is formatted and this then is displayed in the search highlight matching title case displayed on page. Without could display for example `john Smith`

Reply:
doesn't this look weird for prison numbers?

Reply:
True, guess I just thought we don't display prison numbers on the page but maybe this could change in the future, I'll think of an alternative way to do this

Reply:
Removed the titlecase njk filter there